packages/docs/src/stories/templates/table.stories.ts

```diff
@@ -487,7 +544,22 @@ export const TableFixedBottom = {
           });
         });
       </script>
-      <table id="vertical-scrollable-table-2" class="sd-table sample-table h-[200px] block overflow-y-scroll">
+      <table
+        id="vertical-scrollable-table-2"
+        class="sd-table sample-table h-[200px] block overflow-y-scroll focus-visible:focus-outline"
+        role="region"
+        tabindex="0"
+        aria-label="Table with Fixed Bottom Header Row and Shadow"
+      >
+        <thead class="sr-only">
+          <tr>
+            <th>Header</th>
```

Comment:
pls use some real live example for the tabel stories as mentioned in the last "Solution" in the ticket. 

You can also try to use the color tables from the CD-Toolbox (currently broken): https://cd.union-investment.de/grundlagen/Basiselemente/farbe#Primaerfarben

Reply:
I am currently checking if we have a woring version of it.